frontend/src/selectors/staticContentSelectors.js

```diff
@@ -103,27 +109,49 @@ export const getDropdownIncidentFollowupActionOptions = createSelector(
 );
 
 export const getDropdownHSRCMComplianceCodes = createSelector(
-  [getComplianceCodes],
+  [getCurrentComplianceCodes],
   (codes) =>
     codes
       .filter(({ article_act_code }) => article_act_code === "HSRCM")
       .map((code) => {
-        const composedLabel = `${code.section}.${code.sub_section}.${code.paragraph} - ${
-          code.description
-        }`;
+        const composedLabel = code.sub_paragraph
```

Comment:
This could be cleaned up a little to just handle the sub_paragraph directly:
```javascript
`${code.section}.${code.sub_section}.${code.paragraph}${code.sub_paragraph ? `.${code.sub_paragraph}` : ''} - ${code.description}`
```
Or move it outside of the string if the double-interpolation is tough to parse:
```javascript
const subParagraph = code.sub_paragraph ? `.${code.sub_paragraph}` : ''
const composedLabel = `${code.section}.${code.sub_section}.${code.paragraph}${subParagraph} - ${code.description}`
```